src/pages/CreateDao.vue

```diff
@@ -98,7 +98,7 @@
 					name="dao_name"
 					:rules="[{ required: true }]"
 				>
-					<a-input v-model:value="formState.dao_name" />
+					<a-input maxlength="32" v-model:value="formState.dao_name" />
```

Comment:
Quick question.
What will happen here if dao_name is being returned greater then 32 char size? Will it be blank?

Reply:
ok